components/Nav/NavListItem/NavListItem.js

```diff
@@ -41,6 +48,19 @@ export default class NavListItem extends Component {
     this.setState(previousState => ({ areSublinksVisible: !previousState.areSublinksVisible }));
   };
 
+  areSublinksVisible(event) {
+    if (!this.submenu.contains(event.target)) {
```

Comment:
In general, the _only_ time you want to be referencing `this` in our application is to get access to `props`, `state`, or `setState`.

Reply:
What is `submenu` referring to in this situation?